net/https-dns-proxy/files/https_dns_proxy.config

```diff
@@ -1,8 +1,19 @@
 config https_dns_proxy
 	option listen_addr '127.0.0.1'
 	option listen_port '5053'
+	option bind_addr '8.8.8.8,8.8.4.4'
```

Comment:
This name is confusing "-b" doesn't stand for "bind" here. 
These are bootstrap DNS servers that are needed to look up either "dns.google.com" or "cloudflare-dns.com".

Generally "bind" refers to the attaching one end of a socket pair to a local port.

Reply:
I will rename the option. How about `bootstrap_addr`?

Reply:
bootstrap_dns_server? It's long, but 'addr' doesn't really provide any meaning.

Reply:
@aarond10 what about a short "bootstrap_dns", "s" in dns stands for system/service/server etc.

Reply:
`bootstrap_dns` is great! Another alternative would be `fallback_dns`, although unlike Dirk's suggestion that may require some explaining. What do you think Aaron?

Reply:
bootstrap makes more sense given this is used to resolve the hostname for "dns.google.com" or "cloudflare-dns.com", not as any sort of fallback. Happy with the shorter "bootstrap_dns".